jina/types/document/__init__.py

```diff
@@ -498,6 +501,14 @@ def embedding(self, value: Union['np.ndarray', 'jina_pb2.NdArrayProto', 'NdArray
         """
         self._update_ndarray('embedding', value)
 
+    def _update_sparse_ndarray(self, k, v, sparse_cls):
+        protbuf_updater = NdArray(
```

Comment:
To avoid naming conflict, we can directly do

```suggestion
       NdArray(
            is_sparse=True,
            sparse_cls=sparse_cls,
            proto=getattr(self._pb_body, k),
        ).value = v
```

Reply:
Updated removing the local variable